### PR TITLE
Update key-vault.md to keep same consistency with storage.md of .NET

### PR DESCRIPTION
**Pull request by XuGuang-Yao**
Update the Key Vault index pages on docs.microsoft.com for JS with reference to the link
https://docs.microsoft.com/en-us/dotnet/api/overview/azure/storage?view=azure-dotnet

### CODE REVIEW

**Comment by tzhanl** [comment]
@maggiepint @scbedd Please review.

File: docs-ref-services/key-vault.md [hunk 1]
```diff
@@ -14,27 +14,32 @@ ms.service: key-vault
...
+|    keyvault-secrets   |    [Reference](https://docs.microsoft.com/javascript/api/@azure/keyvault-secrets/?view=azure-node-latest)    |    [npm](https://www.npmjs.com/package/@azure/keyvault-secrets)    |    [GitHub](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/keyvault/keyvault-secrets)    |
 
-## Samples
-- Examples for using this module in Node.js as well as browser applications can be found in the [README for the module](https://www.npmjs.com/package/@azure/arm-keyvault)
-- [Getting started with Key Vault in JavaScript](https://azure.microsoft.com/resources/samples/key-vault-node-getting-started/)
-- [Manage Azure resources and resource groups with JavaScript](https://azure.microsoft.com/resources/samples/resource-manager-node-resources-and-groups/) 
-- [Integrating Azure AD into a NodeJS web application](https://azure.microsoft.com/resources/samples/active-directory-node-webapp-openidconnect/) 
+### Version 1.x
```
Comment (heaths) [inline docs-ref-services/key-vault.md:37]: But that package below appears to be the management package (ARM), and should have a different section header. See .NET for example (IIRC, has "Management" in the title).

Comment (heaths) [reply docs-ref-services/key-vault.md:37]: See https://github.com/Azure/azure-docs-sdk-dotnet/pull/1175/files#diff-80809409a19a2a817c014a0770202dceR45

Comment (maggiepint) [reply docs-ref-services/key-vault.md:37]: Also, we should have a table for the v3 packages, along with links to them in the ref

Comment (maggiepint) [reply docs-ref-services/key-vault.md:37]: The right structure would be a 4.x.x table, a 3.x.x table and a management table

**Comment by heaths** [comment]
Also, you might want to fix the PR title (valut -> vault) since it will be the commit message unless you change it before you merge.

File: docs-ref-services/key-vault.md (hunk 1 above)
Comment (XuGuang-Yao) [reply docs-ref-services/key-vault.md]: I can't find the 3.x.x version for key vault service in this repo. @heaths , could you help me on this?

**Comment by maggiepint** [comment]
On this one too - can we change the file name to key-vault-index ?

**Comment by maggiepint** [comment]
Can we add a table with links to the previous version?

**Comment by XuGuang-Yao** [comment]
Hi @maggiepint ,
We can't find the source code and reference for the old version of keyvault.Do you have any idea or suggestion?

**Comment by heaths** [comment]
These are just the JS packages, but maybe there weren't older ones? @sadasant?

**Comment by XuGuang-Yao** [comment]
@heaths , any ideas?

**Comment by scbedd** [comment]
@XuGuang-Yao I can assist with this. You're fine. Let me add a redirect for that file.

**Comment by scbedd** [comment]
Oh. this is the same issue that @DuncanmaMSFT already merged. Thank you Duncan.

**Comment by scbedd** [comment]
Specifically, resolved by these PRS:  [master](https://github.com/MicrosoftDocs/azure-docs-sdk-node/commit/fd64ca0501452b7aa074a6a5075441cf9ddfbc3a)
[Live](https://github.com/MicrosoftDocs/azure-docs-sdk-node/pull/778)

**Pull request merged**